documentation/plain_rules.md

```diff
@@ -969,3 +985,82 @@ A rule that will check if the employee has been hired in the last half hour, cou
     }
 }
 ```
+
+## JSON and Array fields
+
+Some attributes like JSON and Array based, will generate a pseudo-attribute with the
+same name as the attribute and a suffix "\_\_" followed by element name (for the case of JSON) or the ordinal (for the case of arrays), with the parsed value. This value makes easier to write the EPL text which involves time comparisons. 
+
+So, an incoming notification like this:
+
+```json
+{
+    "subscriptionId": "51c04a21d714fb3b37d7d5a7",
+    "data": [
+        {
+            "id": "John Doe",
+            "type": "employee",
+            "myJsonValue": {
+                "type": "myType1",
+                "value": { "color": "blue" }
+            },
+            "myArrayValue": {
+                "type": "myType2",
+                "value": [ "green", "blue" ]
+            }                       
+        }
+    ]
+}
+```
+
+
+which NGSIv1 equivalent is:
```

Comment:
Maybe we could even remove the NGSIv1 example if we want to push the usage of NGSIv2.

Reply:
Fixed in d36021e